armi/utils/directoryChangers.py

```diff
@@ -220,6 +220,8 @@ def __init__(
         dumpOnException=True,
         clean=False,
     ):
+        if not destination:
+            raise ValueError("Non-empty destination directory must be provided.")
```

Comment:
This doesn't appear to check if its empty or not. Consider checking if its empty or changing wording to `valid destination directory must be provided`

Reply:
I meant "empty" as in `""`, not that the folder itself is empty. Will clarify.